_data/projects/OpenTemplate.yml

```
@@ -0,0 +1,8 @@
+name: open template
+desc: boilerplate for a new open source idea
+site: https://github.com/opentemplate/opentemplate
+tags:
+  - boilerplate
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/opentemplate/opentemplate
```

:art: this is the deep-link that goes directly to the list of issues:

```suggestion
  link: https://github.com/OpenTemplate/OpenTemplate/labels/good%20first%20issue
```